css/styles.css

```diff
@@ -146,3 +159,141 @@ li > p {
 .vector {
   height: 30px;
 }
+
+/* Mobile version */
+.grid-container {
+  display: grid;
+  grid-template-columns: auto;
+  background: #191765;
+}
+
+.grid-item {
+  align-self: auto;
+  text-align: center;
+}
+
+.element {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 0;
+}
+
+.work {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #a2a3ae;
+}
+
+.recent-work {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #9490ff;
+}
+
+.page {
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.page1 {
+  background: url(../images/ImgPage1.png) left no-repeat;
+  height: 800px;
+}
+
+.page2 {
+  height: 729px;
+  background: url(../images/ImgPage2.png) right no-repeat;
+}
+
+.page3 {
+  background: url(../images/ImgPage3.png) left no-repeat;
+  height: 729px;
+}
+
+.page4 {
+  background: url(../images/ImgPage4.png) right no-repeat;
+  height: 760px;
+}
+
+.imgPlace {
+  width: 327px;
+  height: 236px;
+  margin-top: 50px;
+}
+
+.block {
+  display: flex;
+  flex-direction: column;
+}
+
+.block p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: justify;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.block ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  height: 50px;
+}
+
+.block ul li {
+  text-align: center;
+  flex-grow: 1;
+  position: relative;
+  border-right: 1px solid #81818c;
+}
+
+.block ul li:last-child {
+  border-right: none;
+}
+
+.block ul li p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+button {
+  background: #7d73fc;
+  color: #edecff;
+  width: 89px;
+  height: 48px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 24px;
+  margin-top: 20px;
+  margin-left: 24px;
+}
```

Comment:
- [ ] Kindly remove the border from the buttons, you can use `border: 0`, to be like the Figma design. 

![image](https://user-images.githubusercontent.com/35535130/154599023-7e4830d0-965e-42cc-bba5-64ee2ee6f69f.png)